Sources/Cocoa/Protocols/StringRepresentable.swift

```diff
@@ -182,6 +184,9 @@ extension UITextView {
         switch string.stringSource {
             case .string(let string):
                 text = string
+                if containsPhoneNumber {
+                    accessibilityValue = accessibilityValue?.replacing("-", with: " ")
+                }
```

Comment:
@emiclark Interesting approach. However, `setText` is widely used in the app and `containsPhoneNumber` doesn't have anything to do with the purpose of the method. Let's revisit this approach on Monday.

Reply:
ok.